CONTRIBUTING.md

```diff
@@ -1,16 +1,29 @@
 # Contributing Guide
 
-To contribute you will need to ensure you have the following setup:
+## Setting up a development environment
 
-- You have a working [Docker environment](https://docs.docker.com/engine).
-- You have a working [Ruby environment](https://ruby-doc.org).
+### Using nix
```

Comment:
I don't think `Using nix` should be the first option listed in `Setting up a development environment`. It implies that it's the recommended and supported one, while I believe currently we still recommend and support the Vagrant option?

Reply:
Sure, I can move it.